opm/test_util/summaryRegressionTest.cpp

```diff
@@ -31,6 +31,8 @@ void RegressionTest::getRegressionTest(){
         OPM_THROW(std::invalid_argument, "Different ammont of keywords in the two summary files.");
     }
 
+    if(printKeyword){printKeywords();}
```

Comment:
If you insist on using one line, you may drop the braces (and add some spacing).